examples/HelloWorldExampleDS/HelloWorldSubscriber.h

```diff
@@ -22,13 +22,10 @@
 
 #include "HelloWorldPubSubTypes.h"
 
-#include <fastrtps/fastrtps_fwd.h>
-#include <fastrtps/attributes/SubscriberAttributes.h>
-#include <fastrtps/subscriber/SubscriberListener.h>
-#include <fastrtps/subscriber/SampleInfo.h>
-
-
-
+#include <fastdds/rtps/attributes/ReaderAttributes.h>
+#include <fastdds/dds/subscriber/DataReaderListener.hpp>
+#include <fastdds/dds/subscriber/Subscriber.hpp>
+#include <fastdds/dds/subscriber/SampleInfo.hpp>
```

Comment:
I am missing some header files being used:

* DomainParticipant
* DataReader

Probably this also happen in the other two header files. Please, check.